draft-ietf-mops-streaming-opcons.md

```diff
@@ -142,14 +178,8 @@ important to consider the effects of network design decisions
 on application-level performance, with considerations for
 the impact on video delivery.
 
-This document aims to provide a taxonomy of networking issues as
-they relate to quality of experience in internet video delivery.
-The focus is on capturing characteristics of video delivery that
-have surprised network designers or transport experts without
-specific video expertise, since these highlight key differences
-between common assumptions in existing networking documents and
-observations of video delivery issues in practice.
-
+This document aims to provide a taxonomy of networking issues as they relate to quality of experience in internet-based streaming video delivery.  The focus is on capturing characteristics of video delivery that have surprised network designers or transport experts without specific video expertise, since these highlight key differences between common assumptions in existing networking documents and observations of video delivery issues in practice.
+ 
```

Comment:
Looks like nothing changed here but whitespace. To keep the diffs meaningful, I usually put each sentence in-full on its own line. Kramdown will combine consecutive lines into one paragraph, but changing a single sentence won't result in large meaningless diff.

Reply:
I think there was a minor wording change to the scope ("internet-based streaming video delivery" instead of "internet video delivery", IIRC).
My intent was to collapse this like you were saying to single lines per paragraph which seems to be best practice, but maybe these should be one big whitespace-only change first.

Reply:
I do a single line per sentence so I can insert or remove sentences without needing to parse the diff for an entire paragraph. Word wrap is definitely worse than either of those approaches, however. :-)

Yes, I would do a whitespace-only PR independently of everything else to get the source formatted the way you want.

Reply:
sure, that's a good idea.  I didn't realize that's what was going on somewhere, but I'll adopt that approach going forward I think. I'll open an issue to normalize the approach.